Documentation/getting_start.md

```diff
@@ -0,0 +1,96 @@
+# Getting start
+
+## 🚀 Deploy the yurt-edgex-manager
+This section tell you how to deploy the yurt-edgex-manger to a cluster
+
+### 👟 Prepare a kubernetes cluster
+To deploy the yurt-edgex-manager alone, you do not need an OpenYurt cluster, but you need deploy the yurt-app-manager.
```

Comment:
"you do not need an OpenYurt cluster"  IMHO, users need an OpenYurt cluster although them can start with creating a Kubernetes cluster and then install OpenYurt components and creating NodePools.